cython/core/fundamental_group.pyx

```diff
@@ -222,6 +223,42 @@ cdef class CFundamentalGroup(object):
     def generators(self):
         """
         Return the letters representing the generators in the presentation.
+	
+	>>> M = Manifold('9_42')
+        >>> G = M.fundamental_group() #Presentation simplified by default
+	>>> G
+        Generators:
+        a,b
+        Relators:
+        aaaabbABBBAbb
+        >>> H = M.fundamental_group(False,False,False) #Unsimplified presentation
```

Comment:
M.fundamental_group(False)

since the other arguments are irrelevant when not simplifying.